vpc/service/config.go

```diff
@@ -86,9 +95,86 @@ func NewConfig(ctx context.Context, v *viper.Viper) (*Config, error) {
 		WorkerRole: v.GetString(config.WorkerRoleFlagName),
 
 		ZipkinURL: v.GetString(config.ZipkinURLFlagName),
+
+		DynamicConfigURL: v.GetString(config.DynamicConfigURLFlagName),
 	}, nil
 }
 
+// Dynamic configs have a default value, which can be overriden at runtime.
```

Comment:
I don't think we use this code actually,
But there is

https://github.com/Netflix/titus-executor/tree/master/properties

And then there is also this?

https://github.com/Netflix/quitelite-client-go/tree/master/properties

Is one or both of those useful to you?

Reply:
I looked at this library. I think you're right, it doesn't seem to be used anywhere. And looks like it was designed to use for getting configs for CLI application built with`cli/altsrc` (e.g. `fetchQuiteLiteSource` returns a `altsrc.InputSourceContext`). I suppose we could modify it to make it work with a more general use case and use it in VPC service. But I personally feel it's easier for us to add some minimal code that just works for VPC service for now.